src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts

```diff
@@ -138,10 +112,7 @@ export class FootprintPolylineVisualizer extends BaseFootprintsVisualizer {
 	onSelectFeature($event) {
 		if ($event.selected.length > 0) {
 			const id = $event.selected[0].getId();
-			const hoverFeature = this.hoverLayer.getSource().getFeatureById(id);
-			if (!hoverFeature || hoverFeature.getId() !== id) {
```

Comment:
check that you won't send same events over and over

Reply:
Seem on-kay!